<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editDialog</class>
 <widget class="QDialog" name="editDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#editDialog{background-image: url(:/res/backtee.jpg);}</string>
  </property>
  <widget class="QGroupBox" name="groupBox_51">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>771</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:transparent;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#ffffff;&quot;&gt;任务配置&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="showOffer">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>20221120-020103-0102</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editCancel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>97</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(56, 56, 56);
color: rgb(200, 200, 200);</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editSave">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>360</y>
      <width>97</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(56, 56, 56);
color: rgb(200, 200, 200);</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,5">
     <property name="spacing">
      <number>30</number>
     </property>
     <item>
      <widget class="QLabel" name="label_327">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;优先级&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="editPriority">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(55, 111, 167);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>优先</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>最高</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>顺序</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,5">
     <property name="spacing">
      <number>30</number>
     </property>
     <item>
      <widget class="QLabel" name="label_378">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;修改起点&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_209">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(55, 111, 167);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>不修改</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>修改</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,5">
     <property name="spacing">
      <number>30</number>
     </property>
     <item>
      <widget class="QLabel" name="label_379">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;修改起点&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_210">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(55, 111, 167);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>不修改</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>修改</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>210</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,3,3">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QLabel" name="label_377">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;新设起点&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_207">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(55, 111, 167);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>优先</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>不优先</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_208">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(55, 111, 167);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>01</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>02</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>270</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,3,3">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QLabel" name="label_380">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;新设终点&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_211">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(55, 111, 167);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>优先</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>不优先</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_212">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(55, 111, 167);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>01</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>02</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>editSave</sender>
   <signal>clicked()</signal>
   <receiver>editDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editCancel</sender>
   <signal>clicked()</signal>
   <receiver>editDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
